src/lotus/message/tests.rs

```diff
@@ -31,8 +31,8 @@ fn test_ipc_address_from_map() {
     {
         "From": {
             "SubnetId": {
-                "Parent": "/r123/f01",
-                "Actor": "f064"
+                "Root": "123",
```

Comment:
Are you sure this is a string? I would have thought that this is a `u64` from the type of the field in `SubnetID`?

Reply:
actually the deserialization can handle both cases, but I changed them to number.